### ISSUE TITLE
django-scribbler is now not installable from source

### ISSUE THREAD

**Issue by imposeren** [caktus/django-scribbler#89]
side libraries like require.js ans codemirror were removed in f194cd16ea4f142a2c40440827a253fa2387126f. This change made it imposible to use djagno-scribbler after installing directly from github: not all js files are available

**Comment by mlavin** [comment]
Yes that was a purposeful choice. There are docs on installing an unstable release here http://django-scribbler.readthedocs.org/en/latest/contributing.html#installing-an-unstable-release

**Comment by imposeren** [comment]
Thank you for information

**Comment by imposeren** [comment]
Maybe this should be added to setup.py to make it easier to install with ``pip install https://github.com/caktus/django-scribbler.git``?

It can be implemented like so:
http://stackoverflow.com/questions/1754966/how-can-i-run-a-makefile-in-setup-py

**Comment by mlavin** [comment]
I'm an -0 on the idea. Building the CSS and JS required LESS and RequireJS which would still need to be present before you could run the setup.py.

**Comment by imposeren** [comment]
> I'm -0 on the idea. Building the CSS and JS required LESS and RequireJS which would still need to be present before you could run the setup.py.

if someone trying to install django-scribbler from git then it's better to fail install with "lessc: command not found" instead of silently installing without required static files

**Comment by mlavin** [comment]
Either way you are going to have to read the docs on how to resolve it. The docs for the current behavior have already been written.

**Comment by imposeren** [comment]
I'm trying to build package locally:
1. installed lessc, requirejs:
```
npm install -g requirejs
npm install -g less
```
2. running:
```
git clone git://github.com/caktus/django-scribbler.git
cd django-scribbler
make fetch-static-libs build-css build-js
```

3. I get error:
```
Tracing dependencies for: scribbler
Error: ENOENT, no such file or directory '.../django-scribbler/scribbler/static/scribbler/libs/underscore.js'
In module tree:
    scribbler

Error: Error: ENOENT, no such file or directory '.../django-scribbler/scribbler/static/scribbler/libs/underscore.js'
```

This happens because underscore url is redirected

**Comment by PetrDlouhy** [comment]
I fixed this in #161
